.gitpod.yml

```diff
@@ -0,0 +1,21 @@
+image:
+  file: .gitpod.Dockerfile
+
+tasks:
+  - init: cargo build --features kvm,virtualbox,xen
+
+vscode:
+  extensions:
+    - editorconfig.editorconfig
+    - rust-lang.rust
+    - matklad.rust-analyzer
+    - eamodio.gitlens
+    - serayuzgur.crates
+    - belfz.search-crates-io
+    - bungcip.better-toml
+
+github:
+  prebuilds:
+    branches: true
```

Comment:
so [`addBadge`](https://www.gitpod.io/docs/prebuilds/#badge) modifies the description a of PR ?
https://www.gitpod.io/docs/prebuilds/#badge
this is really invasive

We can use https://www.gitpod.io/docs/prebuilds/#comment instead to display a button to open Gitpod for a given PR.